YOLO.xcodeproj/project.pbxproj

```diff
@@ -87,11 +77,6 @@
 		63A946D8271800E20001C3ED /* Models */ = {
 			isa = PBXGroup;
 			children = (
-				6381D2132B7817C200ABA4E8 /* yolov8l.mlpackage */,
-				6381D2162B7817C200ABA4E8 /* yolov8m.mlpackage */,
-				6381D2172B7817C200ABA4E8 /* yolov8n.mlpackage */,
-				6381D2152B7817C200ABA4E8 /* yolov8s.mlpackage */,
-				6381D2142B7817C200ABA4E8 /* yolov8x.mlpackage */,
```

Comment:
and here too

Reply:
Same reason as above. If you want to restore it, I will.

Reply:
Keep also here!